discovery/src/main/java/com/ibm/watson/discovery/v2/query/AggregationDeserializer.java

```diff
@@ -1,89 +0,0 @@
-/*
```

Comment:
Does this file need to be deleted? I don't think I see it added back in the hand edits?

Reply:
This is a custom file that was deprecated in previous releases for handling aggregations. So given the way we handle discriminators now in the SDK and because it's a major release I felt safe removing it.